cfpb/cfgov-refresh
# Convert ems to px for mobile size buttons

## Changes

- Convert ems to px for mobile size buttons
- Convert ems to px for search and cta padding to fix vertical alignment.

## Testing

1. pull branch and `gulp clean && gulp build`
2. Visit homepage and resize down to mobile.
3. Expand mega menu hamburger menu and search menu and see that they are the same height. 
4. Go into Settings > Customize fonts and change the font size and re-check the mobile size and see that the menu and search buttons are the same size. Compare to production to see that in production that are virtually unusably large.

## Screenshots

Before:
<img width="658" alt="Screen Shot 2019-07-24 at 3 08 25 PM" src="https://user-images.githubusercontent.com/704760/61821332-ea089f00-ae24-11e9-9733-f187489cf466.png">

After:
<img width="657" alt="Screen Shot 2019-07-24 at 3 08 12 PM" src="https://user-images.githubusercontent.com/704760/61821337-ed9c2600-ae24-11e9-97f9-01fc8111769c.png">
